forge/src/main/java/org/spongepowered/forge/bridge/minecraftforge/fml/RegistryDelegate.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.forge.bridge.minecraftforge.fml;
+
+import org.spongepowered.common.bridge.core.RegistryBridge;
+
+public interface RegistryDelegate<T> {
+
+    RegistryBridge<T> sponge$parent();
+
+    interface Parent<V> extends RegistryBridge<V> {}
```

Comment:
What's the point of this? I can't quite work out what you're trying to do here from the below context. Can you not just use `RegistryBridge<?>` when it's needed?

If you _do_ need a different bridge, make this a different outer interface altogether - this isn't really related to this outer class. `NamespacedWrappedBridge`?

Reply:
Looking back at it with this feedback, the Parent isn't exactly required at all. I originally made this change thinking it'd be a good marker, but I did miss that the extensions already cover RegistryBridge. Is this question reconsidering the whole interface, or just parent?

Reply:
Just the `Parent` interface. I see the point of this delegate.

Reply:
Done, removed parent interface in favor of RegistryBridge